tests/z3-tests.adb

```diff
@@ -813,6 +813,46 @@ package body Z3.Tests is
 
    ---------------------------------------------------------------------------
 
+   procedure Test_Reference_Counting (T : in out Test_Cases.Test_Case'Class)
+   is
+      pragma Unreferenced (T);
```

Comment:
Nit: In other places I started using `Unused_T` as a parameter name which avoids the additional pragma.

Reply:
Fixed in https://github.com/Componolit/AZ3/pull/35/commits/2aac0b6cc800384affd2daebbe6e61db6008163c.